mds/models.py

```diff
@@ -375,6 +375,8 @@ class Compliance(models.Model):
     end_date = models.DateTimeField(blank=True, null=True)
     # Delay considered before computing the compliance
     lag = models.DurationField()
+    # Remember to update this field within raw queries!
```

Comment:
I put this comment everywhere and yet we forgot to update the raw queries. :joy:

Hopefully there are none for the compliances.